Source/.editorconfig

```diff
@@ -0,0 +1,25 @@
+﻿# EditorConfig is awesome: http://EditorConfig.org
+
+# Top-most EditorConfig file
+root = true
+
+[*]
+# 4-column space indentation
+indent_style = space
+indent_size = 4
+
+charset = utf-8
+trim_trailing_whitespace = true
+
+max_line_length = 160
+
+insert_final_newline = true
+
+[*.cs]
+# Windows-style newlines
+end_of_line = crlf

```

Comment:
I would probably move this line into the `[*]` section and kill the `[*.cs]` section. If we ever add any file types which need LF, such as `.sh` then we can add a section for them.